### PR TITLE
[FSSS-178] Adds Changelog file

### PR DESCRIPTION
**Pull request by danzanzini**
## What's the purpose of this pull request?
This PR **adds the CHANGELOG file** along with its related documentation.
For documentation, I've added a **checklist item in the PR template** and created a **new section on the README called `Contributing`** that, for now, only has one item related to the Changelog.

## How it works? 
The CHANGELOG file follows the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention. Each PR should add at least one entry in the `[UNRELEASED]` section of the CHANGELOG. When it's time to release all of these entries are moved to a entry related to the version being released and a new clean `UNRELEASED` section is created to receive entries for new PRs.

I've set a `0.1.0` version on the CHANGELOG that is the closed beta release. We should create a new Github Release version related to this version when we start using our new Github flow

## Jira link
https://vtex-dev.atlassian.net/browse/FSSS-178

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -333,7 +342,7 @@ The aforementioned guide works well for UI components. However, components like
 
 ## 🖊️ Styling Components
 
-Our customized theme is based on [Design Tokens](https://css-tricks.com/what-are-design-tokens/) using [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) or a CSS class for each token. Today, we have the following files in the `src/styles` folder:
+Our customized themes is based on [Design Tokens](https://css-tricks.com/what-are-design-tokens/) using [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) or a CSS class for each token. Today, we have the following files in the `src/styles` folder:
```
Comment (georgebrindeiro) [inline README.md:345]: É um tema ou vários temas?

Comment (georgebrindeiro) [reply README.md:345]: Se forem vários, tem que botar:

```suggestion
Our customized themes are based on [Design Tokens](https://css-tricks.com/what-are-design-tokens/) using [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) or a CSS class for each token. Today, we have the following files in the `src/styles` folder:
```

**Review by filipewl — APPROVED** [review]
LGTM! Maybe we can consider adopting https://www.conventionalcommits.org/en/v1.0.0/ for automated changelogs later.

File: README.md (hunk 1 above)
Comment (hellofanny) [reply README.md:345]: Atualmente só temos um 👀 @renatamottam

Comment (danzanzini) [reply README.md:345]: I'll accept the suggestion since the plan is to have more than one.

**Review by hellofanny — APPROVED** [review]
🚀 🚀 🚀

**Review by ArthurTriis1 — APPROVED** [review]

**Pull request merged**